partner_industry_secondary/migrations/11.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_res_partner_industry(cr):
+    cr.execute(
+        """
+        INSERT INTO res_partner_industry (name, active, create_uid,
+            create_date, write_uid, write_date, %s)
+        SELECT name, TRUE as active, create_uid, create_date, write_uid,
+            write_date, id
+        FROM res_partner_sector
+        """ % openupgrade.get_legacy_name('old_sector_id'),
```

Comment:
You are injecting here SQL values. Proper syntax is:
```suggestion
        """, (AsIs(openupgrade.get_legacy_name('old_sector_id')), ),
```

Reply:
And previously you have to import:
```python
from psycopg2.extras import AsIs
```

Reply:
It's
```python
from psycopg2.extensions import AsIs
```

Reply:
Agh, my memory has betrayed me!